.github/workflows/template-setup-e2e-test/action.yaml

```diff
@@ -4,21 +4,17 @@ inputs:
   kubernetes-version:
     required: true
     type: string
-  cluster_name:
-    required: false
-    type: string
-    default: katib-e2e-cluster
 
 runs:
   using: composite
   steps:
-    - name: Set Up KinD Cluster
-      uses: helm/kind-action@v1.2.0
+    - name: Set Up Minikube Cluster
+      uses: manusa/actions-setup-minikube@v2.6.0
```

Comment:
I used this action to change to Minikube Cluster with minimal changes.
Do you prefer this action or a manual installation?
If you prefer manual installation rather than this action, I will change the way to install Minkube.

Reply:
WDYT? @johnugeorge

Reply:
Can we keep the manual way as kserve? Do you see any red flags?

Reply:
There is no difference in operation between this action and manual installation.
The only difference is whether we manage the scripts or the administrator of this action manages the scripts.